client/src/components/Home/index.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import styled from "styled-components";
+import { spacing64 } from "../../resources/style-constants";
+
+import Invitation from './Invitation';
+import ActionCard from './ActionCard';
+
+const Wrapper = styled.div`
+  padding: ${spacing64};
+`;
+
+const Home = () => {
```

Comment:
I also added a home page component. I can remove this if you have other plans for the home page component.